LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/OTAAHelper/OTAAKeysGenerator.cs

```diff
@@ -10,13 +10,13 @@ namespace LoRaTools
 
     public class OTAAKeysGenerator
     {
+        private static readonly Random RndDevAddr = new Random();

```

Comment:
Better than before, but I think we could have problems using shared object in multiple threads.

https://devblogs.microsoft.com/pfxteam/getting-random-numbers-in-a-thread-safe-way/

https://stackoverflow.com/questions/3049467/is-c-sharp-random-number-generator-thread-safe

Reply:
You thought you would sneak it in this one here..... ah ah.... but no we keep it like this.

Reply:
Ok. Didn't know about the fact that this thing is not thread safe. Doesn't make much sense, as it will be used as a static almost everywhere (if you want to make sure you get unique numbers for sure). I was sceptical that this was still a problem in the current framework implementations (Jave btw. does properly lock it apparently), but I was able to repro and get it into a state where it only returned 0. I'll add a lock to protect from that, but will keep the static.